implementations/micrometer-registry-statsd/src/main/java/io/micrometer/statsd/StatsdMeterRegistry.java

```diff
@@ -87,6 +88,8 @@
     @Nullable
     private Consumer<String> lineSink;
 
+    public static final AttributeKey<Boolean> CONNECTION_DISPOSED = AttributeKey.valueOf("doOnDisconnectCalled");
```

Comment:
May be package private?

Reply:
Thanks for catching this. I made it public by mistake, but there's no need for it as we don't even reference this directly in tests.